src/nvim/getchar.c

```diff
@@ -4261,3 +4261,21 @@ static bool typebuf_match_len(const uint8_t *str, int *mlen)
   *mlen = i;
   return str[i] == NUL;  // matched the whole string
 }
+
+mapblock_T * get_maphash(int index, char *scope)
```

Comment:
`const char *const`. And FUNC_ATTR_NONNULL_ALL FUNC_ATTR_PURE. Also no space after asterisk.